app/views/devise/registrations/edit.html.haml

```diff
@@ -16,9 +16,9 @@
             .col-md-5.col-md-offset-1
               %h3.general-subtitle.mt-0= t('settings.shipping')
       .row
-        = render partial: 'devise/registrations/settings/address', locals: { address: :billing }
-        = render partial: 'devise/registrations/settings/address', locals: { address: :shipping }
+        = render partial: 'settings/address', locals: { address: :billing, resource: GetUsersAddress.call(user: current_user, address: :billing) }
```

Comment:
You have not to call some QueryObject, ServiceObject or Interactor from your views.